projects/schematics/src/migrations/3_0/component-deprecations/data/place-order.component.migration.ts

```diff
@@ -0,0 +1,13 @@
+import { PLACE_ORDER_COMPONENT } from '../../../../shared/constants';
+import { ComponentData } from '../../../../shared/utils/file-utils';
+
+export const PLACE_ORDER_COMPONENT_MIGRATION: ComponentData = {
+  selector: 'cx-place-order',
+  componentClassName: PLACE_ORDER_COMPONENT,
+  removedProperties: [
+    {
+      name: 'placeOrderSubscription',
+      comment: `'placeOrderSubscription' property was removed - no replacement`,
```

Comment:
The name was changed to a more generic name `subscription` and just instantiates a new Subscription object to support other Observables than the placeOrder one

So I don't think it was removed